CHANGELOG.md

```diff
@@ -23,14 +23,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Enable SIMD autodetection for Amazon Voice Focus in Chrome 90+.
 - Clean up task cancel hooks after they cease to be relevant.
 - Enable sender-side bandwidth estimation in Safari.
+- Clean up usage of audioDeviceInformation in ReceiveAudioInputTask.
 
 ### Removed
 
-- Removed audioDeviceInformation from AudioVideoControllerState.
```

Comment:
I think we should keep this line in the removed section (as one of the old PRs is performing this action) and add a new entry for this PR.
It becomes easy for referencing in the future.

Reply:
done